src/main/openapi/ds-discover-openapi_v1.yaml

```diff
@@ -235,6 +235,144 @@ paths:
 
         # TODO: Add more facet variables
 
+
+
+        # Spell check fields
+        - name: spellcheck
+          in: query
+          description: | 
+            Solr spellcheck enabling [spellcheck](https://solr.apache.org/guide/solr/latest/query-guide/spell-checking.html)
+            
+            If true, solr response will contain spellcheck information. Spellcheck will only be activated by Solr if no results are found
```

Comment:
Maybe add a line on what the functionality of spellcheck is. What do you gain by enabling spellcheck

Reply:
add one line